kernelci/api/models.py

```diff
@@ -507,6 +529,12 @@ class Regression(Node):
         description='Type of the object',
         const=True
     )
+    result: Optional[ResultValues] = Field(
+        description=("PASS if the regression is 'inactive', that is, if the "
+                     "test has ever passed after the regression was created. "
+                     "FAIL if the regression is still 'active', ie. the test "
+                     "is still failing")
+    )
```

Comment:
Related to https://github.com/kernelci/kernelci-pipeline/pull/503/files#r1544459852
```suggestion
    result: ResultValues = Field(
        default=ResultValues.FAIL.value,
        description=("PASS if the regression is 'inactive', that is, if the "
                     "test has ever passed after the regression was created. "
                     "FAIL if the regression is still 'active', ie. the test "
                     "is still failing")
    )
```